proxy/src/auth/backend/jwt.rs

```diff
@@ -7,40 +11,34 @@ use jose_jwk::crypto::KeyInfo;
 use signature::Verifier;
 use tokio::time::Instant;
 
-use crate::{http::parse_json_body_with_limit, intern::EndpointIdInt};
+use crate::{context::RequestMonitoring, http::parse_json_body_with_limit, EndpointId, RoleName};
 
 // TODO(conrad): make these configurable.
+const LEEWAY: Duration = Duration::from_secs(30);
```

Comment:
I think this could use a more descriptive name. Up to you. At least the config option should have a good explanation.

Reply:
In the context of JWT, it think it's fairly clear. I'm using the terminology from https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4

Reply:
Renamed to make it clear it's clock-skew related.